src/views/view-log.php

```diff
@@ -26,10 +26,10 @@
             <dd><?= Html::encode($model->schedulerTask->description) ?></dd>
 
             <dt><?= $model->getAttributeLabel('started_at') ?></dt>
-            <dd><?= Yii::$app->formatter->asDatetime($model->started_at) ?></dd>
```

Comment:
What is the reasoning around removing the call to the formatter here? we use this to take advantage of Yiis localised date formatting